doc/overlapping.md

```diff
@@ -0,0 +1,145 @@
+# Overlapping strategy
+
+## Market Price used for calculation
+To calculate the price & budget we need the market price. There are 3 types of market price
+- user market price
+- external market price
+- calculated market price
+
+**User Market Price**
+
+Price set by the user in the UI. It's a search param of the URL `search.marketPrice`.
+
+**External Market Price**
+
+Price given by CoinGecko.
+- If one of the token is not listed: `undefined`
+- If both token are listed: `basePrice / quotePrice`
+
+**Calculated Market Price**
+
+Price calculated from the initial strategy.
+- If no budget: `undefined`
+- If not overlapping: `undefined`
+- If is paused: `undefined`
+- If above market price: `buyMarginalPrice`
+- If below market price: `sellMarginalPrice`
+- Else: `geomean(buyMarginalPrice, sellMarginalPrice)`
+
+### Hierarchy
+The behavior is different if the strategy has been **touched** or not.
+
+**touched**
+
+Price is considered **touched** if : 
+- User market price is set
+- Strategy is not overlapping
+- Strategy has no budget
+- Strategy is paused
+- Min price has been updated
+- Max price has been updated
+- Spread has been updated
```

Comment:
Consider separating some of these conditions between create and edit. Create has all conditions that edit does but edit has 3 extra ones:

```suggestion
Price is considered **touched** if : 
- User market price is set
- Min price has been updated
- Max price has been updated
- Spread has been updated
- Strategy is not overlapping (only for edit)
- Strategy has no budget (only for edit)
- Strategy is paused (only for edit)
```